### PR TITLE
Change slack link in readme to invite based

### PR DESCRIPTION
**Pull request by derberg**
The current link takes you to slack sign-in page, but if it is your first time (like mine) you are doomed. I had to look for a proper invite-like link on a website, so I just want to avoid others follow my struggle.

Also fixed broken link to example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -206,11 +206,11 @@ async function main() {
 main();
 ```
 
-The argument to `quicktype` is a complex object with many optional properties. [Explore its definition](https://github.com/quicktype/quicktype/blob/javascript-use-examples/src/quicktype-core/Run.ts#L171) to understand what options are allowed.
+The argument to `quicktype` is a complex object with many optional properties. [Explore its definition](https://github.com/quicktype/quicktype-vscode/blob/master/src/extension.ts#L141) to understand what options are allowed.
```
Comment (skornel02) [inline README.md:209]: Isn't the definition in quicktype-vscode only a subset of total functionality?
[The main entry point](https://github.com/quicktype/quicktype/blob/master/src/quicktype-core/Run.ts#L119) has more options to offer.

Comment (derberg) [reply README.md:209]: @skornel02 you are absolutely right, I've just learned it today after digging into the project. The learning curve here is pretty steep 😅

Comment (derberg) [reply README.md]: changed

**Comment by schani** [comment]
Thank you! :-)

**Pull request merged**